specs/FrameProcesssInfo.md

```diff
@@ -0,0 +1,540 @@
+                            wil::com_ptr<ICoreWebView2ProcessInfo2> processInfo2;
+                            CHECK_FAILURE(
+                                processInfo->QueryInterface(IID_PPV_ARGS(&processInfo2)));
+                            wil::com_ptr<ICoreWebView2FrameInfoCollection> frameInfoCollection;
+                            CHECK_FAILURE(processInfo2->get_AssociatedFrameInfos(
+                                &frameInfoCollection));
+                            wil::com_ptr<ICoreWebView2FrameInfoCollectionIterator> iterator;
+                            CHECK_FAILURE(frameInfoCollection->GetIterator(&iterator));
+                            BOOL hasCurrent = FALSE;
+                            UINT32 frameInfoCount = 0;
+                            while (SUCCEEDED(iterator->get_HasCurrent(&hasCurrent)) &&
+                                   hasCurrent)
+                            {
+                                wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+                                CHECK_FAILURE(iterator->GetCurrent(&frameInfo));
+
+                                AppendFrameInfo(frameInfo, rendererProcessResult);
+                                AppendAncestorFrameInfo(frameInfo, rendererProcessResult);
+
+                                BOOL hasNext = FALSE;
+                                CHECK_FAILURE(iterator->MoveNext(&hasNext));
+                                frameInfoCount++;
+                            }
+                            rendererProcessResult.insert(0, std::to_wstring(frameInfoCount) +
+                                                L" frameInfo(s) found in Renderer Process ID:" +
+                                                std::to_wstring(processId) + L"\n");
+                            result.append(rendererProcessResult + L"\n");
+                            rendererProcessCount++;
+                            //! [AssociatedFrameInfos]
+                        }
+                        else
+                        {
+                            otherProcessResult.append(L"Process Id:" + std::to_wstring(processId) +
+                                            L" | Process Kind:" + ProcessKindToString(kind) + L"\n");
+                        }
+                    }
+                    result.insert(0, std::to_wstring(processCount) +
+                               L" process(es) found, from which " +
+                               std::to_wstring(rendererProcessCount) + 
+                               L" renderer process(es) found\n\n");
+                    otherProcessResult.insert(0, L"\nRemaining " +
+                                           std::to_wstring(processCount - rendererProcessCount) +
+                                           L" Process(es) Infos:\n");
+                    result.append(otherProcessResult);
+                    MessageBox(nullptr, result.c_str(), L"Process Info with Associated Frames", MB_OK);

```

Comment:
We don't want to do run a message loop within a WebView2 event handler or WebView2 async method completion callback. You should be able to see examples in other of our sample code that runs a lambda asynchronously outside the async method completion callback to avoid this.